Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4837,7 +4837,7 @@ def __init__(self):
             self.http_client_print = getattr(http_client, 'print', None)  # save in case someone else patched it already
             setattr(http_client, 'print', self.int_logger.print_override)
         self.handler = DemistoHandler(self.int_logger)
-        demisto_formatter = logging.Formatter(fmt='%(asctime)s - %(message)s', datefmt=None)
+        demisto_formatter = logging.Formatter(fmt='python logging: %(levelname)s [%(name)s] - %(message)s', datefmt=None)
```

Comment:
why it is important to add `python logging:` prefix?

Reply:
So it is clear in the log that this log line came via python logging. Otherwise it can be mis-leading.